jwst/datamodels/schemas/specprofilesingle.schema.yaml

```diff
@@ -0,0 +1,33 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
+id: "http://stsci.edu/schemas/jwst_datamodel/specprofilesingle.schema"
+title: Spectral profile data
+allOf:
+- $ref: referencefile.schema
+- $ref: keyword_exptype.schema
+- $ref: subarray.schema
+- type: object
+  properties:
+    data:
+      title: The profile data
+      fits_hdu: ORDER
+      max_ndim: 2
+      datatype: float32
+- type: object
+  properties:
```

Comment:
If the following three are going directly under `tree` and not under `meta` there's no need of another `object` definition.
And the same for the rest of the files.

Reply:
Oh, OK. I'll try making those changes.

Reply:
Done